MigrationV3V4/Node/core-MultiDialogs-v4/const.js

```diff
@@ -9,5 +9,7 @@ module.exports = {
     FLIGHTS_DIALOG: 'flightsDialog',
     SUPPORT_DIALOG: 'supportDialog',
     USER_PROFILE_PROPERTY: 'userProfileProperty',
-    CONVERSATION_STATE_ACCESSOR: 'conversationStateAccessor'
+    CONVERSATION_STATE_ACCESSOR: 'conversationStateAccessor',
+    HOTELS_WATERFALL_DIALOG: 'hotelsWaterfallDialog',
```

Comment:
I'm not sure why you'd make the two _waterfall dialog IDs_ public constants. Each will be local within their respective component dialog, and nothing outside the component dialog will be able to make relevant use of it.

Reply:
agree...removed consts